StripeUICore/StripeUICore/Source/Controls/OneTimeCodeTextField.swift

```diff
@@ -62,18 +51,38 @@ final class OneTimeCodeTextField: UIControl {
     }
 
     private lazy var digitViews: [DigitView] = (0..<numberOfDigits).map { _ in
-        return DigitView()
+        return DigitView(theme: theme)
     }
 
-    override var canBecomeFirstResponder: Bool {
-        return true
+    private let feedbackGenerator = UINotificationFeedbackGenerator()
+
+    // MARK: - UIKeyInput properties
+
+    public var keyboardType: UIKeyboardType = .asciiCapableNumberPad
+
+    public var textContentType: UITextContentType? = .oneTimeCode
+
+    // MARK: - UITextInput properties
+
+    public var selectedTextRange: UITextRange? {
+        willSet {
+            inputDelegate?.selectionWillChange(self)
+        }
+        didSet {
+            inputDelegate?.selectionDidChange(self)
+            update()
+        }
     }
 
-    private let feedbackGenerator = UINotificationFeedbackGenerator()
+    public var inputDelegate: UITextInputDelegate?
+
+    public lazy var tokenizer: UITextInputTokenizer = UITextInputStringTokenizer(textInput: self)
 
-    init(numberOfDigits: Int = 6) {
+    // MARK: -
+    public init(numberOfDigits: Int = 6, theme: ElementsUITheme = .default) {
```

Comment:
looking at: `theme: ElementsUITheme = .default`

if you open Link right now in the example app, does it show the coloring you'd expect? 

just double-checking that since I guess this was hard-coded to Link color, that it still has the colors one would expect

Reply:
It does! The Link call sites have all been updated to use the LinkUI theme, and (other than the slight text color change) it looks as it previously did in all instances :)